aten/src/ATen/native/Resize.cpp

```diff
@@ -21,15 +22,29 @@ Tensor& resize_cpu_(Tensor& self, IntArrayRef size) {
 // this function in native_functions.yaml
 Tensor& resize_as_sparse_(Tensor& self, const Tensor& src);
 
-Tensor& resize_as_(Tensor& self, const Tensor& the_template) {
+Tensor& resize_as_(
+    Tensor& self,
+    const Tensor& the_template,
+    c10::optional<MemoryFormat> optional_memory_format) {
   if (self.is_sparse() && the_template.is_sparse()) {
+    TORCH_CHECK(
+        !optional_memory_format.has_value(),
+        "Unsupported memory format for sparse tensor resize_as_ :",
+        optional_memory_format.value_or(MemoryFormat::Contiguous));
```

Comment:
can't you use `optional_memory_format.value()` or `*optional_memory_format`, because you know it has a value?